theme/icons.go

```diff
@@ -40,6 +40,30 @@ func NewThemedResource(dark, light fyne.Resource) *ThemedResource {
 	}
 }
 
+// FocusedResource is a resource wrapper that will return an appropriate resource colorized by
+// the current theme's `FocusColor` color.
+type FocusedResource struct {
+	source fyne.Resource
+}
+
+// Name returns the resource source name prefixed with `focused_` (used for caching)
+func (res *FocusedResource) Name() string {
+	return fmt.Sprintf("focused_%s", res.source.Name())
+}
+
+// Content returns the disabled style content of the correct resource for the current theme
+func (res *FocusedResource) Content() []byte {
+	clr := fyne.CurrentApp().Settings().Theme().FocusColor()
```

Comment:
This is what I mentioned in the comment - focusColor assumed highlighting a background coloured item not foreground so it is currently too dark for this new usage... Let's chat :)